server/controllers/userController.js

```diff
@@ -36,6 +43,50 @@ const emailValidator = (req, res, next) => {
     next();
   }
 };
+// check picture is safe or not
+const profileImageValidator = async (imageBuffer) => {
```

Comment:
I think this should be moved to its own file with a generic name so that we can re-use it for other purposes like (e.g #422)

Reply:
can you plz tell should I create new folder name or just add new file in the controllers?

Reply:
You can create a new file in the `utils` directory

Reply:
@hubsMIT1 don't do this change request

Reply:
Is my PR ready to be merged now?